### PR TITLE
add: add the usage about Radxa Zero

### PR DESCRIPTION
**Pull request by Kinvy66**
Add the usage about Radxa Zero，include Chinese and Enlish.
Most of the content is on the wiki, I just shipped it to doc websit and translate it.

### CODE REVIEW

**Comment by Kinvy66** [comment]
`Description of changes`
Add the usage about Radxa Zero，include Chinese and Enlish.
Most of the content is on the wiki, I just shipped it to doc websit and translate it.

`Review guide`
To review this PR locally, please run the following commands:
```
PR= # the current PR number
git fetch origin pull/${PR}/head
git switch --detach FETCH_HEAD
yarn start
yarn start --locale en
# If you want to start 2 locales side-by-side, use the following command on Linux:
# https://github.com/facebook/docusaurus/issues/7377#issuecomment-1167192715
yarn start & DOCUSAURUS_GENERATED_FILES_DIR_NAME=.docusaurus/en yarn start --locale en --port 3001
kill %1
```

File: docs/zero/zero/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+#### Radxa Zero 开发板
+
+#### 存储设备:
+
+- 板载的eMMC 或
+- 如果开发板没有板载eMMC需要准备一个至少8GB的sd卡. eMMC是板子背面的那个大的贴片芯片，你可以检查下你的板子是否有这个元件.<br/>如果需要安装带有桌面环境的镜像，请使用至少16GB的sd卡，最好是32GB。
+
+#### USB-C hub
+
+- Radxa Zero 有一个板载 Type-C USB3.0接口（靠近板子中间的那个USBC口）。这个接口支持USB3.0 Type-A和有线网，但是不具备视频输出功能。
```
Comment (jack-ma) [inline docs/zero/zero/getting-started.md:20]: 改为如下更合适：

这个接口支持通过 USB Type-C hub 转出 Type-A 口和有线网(如果 hub 支持)，但是不具备视频输出功能。

File: docs/zero/zero/maskrom.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+在windows中的设备管理器，可以看到出现下面的设备
+![Windows-GX-CHIP](/img/zero/zero/Windows-GX-CHIP.webp)
+
+## 安装驱动
+
+下载安装[Zagdig](https://zadig.akeo.ie/)软件安装USB驱动，如果在插入设备后没有出现下图中的设备信息，选择 `Options`->`List All Devices`
+
+![Zagdig-libusb](/img/zero/zero/Zagdig-libusb.webp)
+
+Windows下需要使用[Radxa DL](https://dl.radxa.com/zero/tools/windows/RZ_USB_Boot_Helper_V1.0.0.zip)软件从USB加载二进制文件，比如擦除eMMMC的操作。连接Radxa Zero进入Maskrom, Radxa DL **没有**显示 `Not detected, press and hold USB boot key and power on` 的提示就表示已经进入maskrom。接下来就可以进行[擦除eMMC](/zero/zero/erase-emmc).
```
Comment (jack-ma) [inline docs/zero/zero/maskrom.md:31]: [Radxa DL] 需要改为 [RZ USB Boot Helper 工具]

File: i18n/en/docusaurus-plugin-content-docs/current/zero/zero/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+### Required
+
+- Radxa Zero main board
+- Storage media:
+  - Built-in eMMC module, or
+  - A microSD card that is at least 8GB if your board doesn't have an eMMC module. You can confirm this by checking if there is a big chip soldered on the bottom side of the board.
+    For desktop usage we recommended at least 16GB, preferably 32GB.
+- USB-C hub
+  - Radxa Zero comes with one USB 3.0 Type-C port, located on the top side and is close to the center of the board. This port can be expanded into multiple USB 3.0 Type-A ports as well as Ethernet.No HDMI or DisplayPort Alternate Mode was implemented on this port, so you won't get video output even if your USB-C hub has those connectors.
```
Comment (jack-ma) [inline i18n/en/docusaurus-plugin-content-docs/current/zero/zero/getting-started.md:17]: add "via USB Type-C Hub" at the end of as well as Ethernet

File: docs/zero/zero/erase-emmc.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 30
```
Comment (jack-ma) [inline docs/zero/zero/erase-emmc.md:2]: 擦除 eMMC 的顺序应该在 Maskrom 模式之后，需要调整 `sidebar_position`

**Comment by jack-ma** [comment]
Thanks for your PR. I commented some small issues, please change it and push another commit directly to your branch, we will merge it after your fix.

**Comment by Kinvy66** [comment]
I have fix the issues, and pushed it.

**Comment by u7985** [comment]
Hi Kinvy66,

Thanks for your work. But this PR contains other commits and changes many files. It looks like it might need to close and then open a new PR.

**Comment by Kinvy66** [comment]
> Hi Kinvy66,
> 
> Thanks for your work. But this PR contains other commits and changes many files. It looks like it might need to close and then open a new PR.


I'm not familiar with GitHub's PR process. I performed a synchronization operation before the second cmmit of the this PR. which resulted in multiple user commits for this PR. I submitted a PR [#410](https://github.com/radxa-docs/documentation/pull/410) again, and the new content is the use of Zero board.

**Pull request closed without merging**